src-plugins/medSegmentation/msegAlgorithmPaintToolbox.cpp

```diff
@@ -978,11 +981,7 @@ void AlgorithmPaintToolbox::updateWandRegion(medAbstractImageView * view, QVecto
     }
 
     if(!view->contains(m_maskAnnotationData))
-    {
         view->addLayer(m_maskAnnotationData);
-        setOutputMetadata(m_imageData, m_maskData);
-        medDataManager::instance()->importData(m_maskData);
-    }
```

Comment:
on my side, it was not my mask. You begin to "paint" on the data, and after one second, the software freezes because it's saving a data with just your beggining of ROI, i.e. trash.
:)